modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -38,10 +39,18 @@ func NewKeeperWithVM(
 	authority string,
 	vm *wasmvm.VM,
 ) Keeper {
+	if vm == nil {
+		panic("wasm VM must be not nil")
+	}
+
 	if types.WasmVM != nil && !reflect.DeepEqual(types.WasmVM, vm) {
 		panic("global Wasm VM instance should not be set to a different instance")
 	}
 
+	if strings.TrimSpace(authority) == "" {
+		panic("authority must be non-empty")
+	}
```

Comment:
should we panic with errors instead? then we test using `suite.RequirePanicsWithError`?

I guess it doesn't really make a difference as a panic is a panic, but seems like it makes sense to panic with an error. Afaik other `NewKeeper` funcs are doing this atm, although its probably not 100% consistent across the board.

Reply:
Indeed, I think it's not consistent in the codebase, but I can for now change that here.